xml/gl.xml

```diff
@@ -268,13 +268,13 @@ typedef unsigned int GLhandleARB;
     </enums>
 
     <enums namespace="GL" group="TraceMaskMESA" type="bitmask">
-        <enum value="0x0001" name="GL_TRACE_OPERATIONS_BIT_MESA" group="TraceMaskMESA"/>
```

Comment:
Why has this group been removed? It has not been replaced with an alternative group.

Reply:
Ah, I didn't find this back then:
https://opensource.apple.com/source/X11server/X11server-85/mesa/Mesa-7.0.4/docs/MESA_trace.spec.auto.html
Well, I've added this group back (and also 2 other enums to `GetPName` and `StringName`).
But there is nowhere to use them, because only enums of this extension exist, not functions, so I didn't push this.

I don't think it makes sense to have enums in any binding, when that binding doesn't also expose functions to use them these enums.
If bindings user needs to manually add functions - they should handle related enums too.
So the way to go should be either to add functions and a .txt file to the registry (if they are usable, even if outdated) or remove the enums.

Or, another alternative, if the point of adding enums and only them was to reserve points in enums space, it should instead be done with `<unused/>` tag.